core/tests/protractor_desktop/topicAndStoryEditor.js

```diff
@@ -304,12 +304,13 @@ describe('Chapter editor functionality', function() {
       browser.get('/create/' + dummyExplorationIds[0]);
       waitFor.pageToFullyLoad();
       explorationEditorMainTab.setContent(function(richTextEditor) {
-        richTextEditor.addRteComponent('Skillreview', 'Description');
+        richTextEditor.addRteComponent(
+          'Skillreview', 'Description', 'skillFromChapterEditor0');
       });
       explorationEditorPage.navigateToPreviewTab();
       explorationPlayerPage.expectContentToMatch(function(richTextChecker) {
         richTextChecker.readRteComponent(
-          'Skillreview', 'Description', forms.toRichText('material1'));
```

Comment:
Why did this change from material1 to material0?

Reply:
Initially, without specifying a skill to select, the test was selecting the second skill, but now since I am specifying the skill to select to be 'skill0', the review material is tested against that of the first skill.